tests/query_tests.rs

```diff
@@ -24,23 +25,34 @@ fn check_compilation(
     // This is the binary to compile a query
     let mut cmd = Command::new(proj_dir.join("target/debug/dtc"));
     // Assemble the args, first the input query
-    let mut args = vec!["-q".to_string(), format!("{}", query_file.display()).into()];
+    let mut args = vec!["-q".to_string(), query_file.to_str().unwrap().into()];
     // Append every udf that is provided
     for udf_name in udf_names {
         let udf_file = udf_dir.join(udf_name);
         args.push("-u".to_string());
-        args.push(format!("{}", udf_file.display()).to_string());
+        args.push(udf_file.to_str().unwrap().into());
     }
     let mut out_file = query_dir.join(query_name);
     out_file.set_extension("cc");
     args.extend(vec!["-o".to_string(), out_file.to_str().unwrap().into()]);
+    args.push("-rproductpage-v1".to_string());
```

Comment:
Shouldn't there be a space between -r and productpage-v1?

Reply:
Apparently, this is how rust parses the input. If there is a space, there will also be a space in the string. There is probably a way to fix this.

Reply:
Huh.  Well, we should probably put a TODO in main, because it won't be fun for any new users, but if it works for now, let's just go with it.